app/src/components/nav/ProjectBreadcrumbContents.tsx

```diff
@@ -0,0 +1,34 @@
+import { HStack, Flex, Text } from "@chakra-ui/react";
+import Link from "next/link";
+import { useSelectedOrg } from "~/utils/hooks";
+
+// Have to export only contents here instead of full BreadcrumbItem because Chakra doesn't
+// recognize a BreadcrumbItem exported with this component as a valid child of Breadcrumb.
+export default function ProjectBreadcrumbContents({ orgName = "" }: { orgName?: string }) {
+  const { data: selectedOrg } = useSelectedOrg();
+
+  orgName = orgName || selectedOrg?.name || "";
+
+  return (
+    <Link href="/home">
+      <HStack w="full">
+        <Flex
+          p={1}
+          borderRadius={4}
+          backgroundColor="orange.100"
+          minW={6}
```

Comment:
I am confused don't these 6 values together do the same thing as `boxSize={6}`?

Reply:
Yes they do! I didn't realize boxSize could be used on anything besides icons.